app/models/attachment.rb

```diff
@@ -0,0 +1,23 @@
+require "gds_api/asset_manager"
+
+class Attachment < Document
+  attr_accessor :title, :file, :content_type, :url, :content_id, :created_at, :updated_at, :has_changed
+
+  def initialize(params={})
+    @title = params[:title]
+    @file = params[:file]
+    @content_type = params[:content_type]
+    @url = params[:url]
+    @content_id = params[:content_id] || SecureRandom.uuid
+    @created_at = params[:created_at]
+    @updated_at = params[:updated_at]
+    @has_changed = false
+  end
+
+  def update_attributes(new_params)
+    new_params.each do |k, v|
```

Comment:
Should these params be restricted in some way? Would throw a NoMethodError (or worse) if something unexpected was passed in?